<launch>
  <arg name="bag" />
  <arg name="param" />

  <!-- SVO --> 
  <node pkg="svo_ros" type="svo_node" name="svo" clear_params="true" output="screen">
    <!-- Camera and IMU topics to subscribe to -->
    <param name="cam0_topic" value="/StereoImage/left" type="str" />
    <param name="cam1_topic" value="/StereoImage/right" type="str" />
    <param name="imu_topic" value="/IMU/data_raw" type="str" />
    <param name="calib_file" value="$(find svo_ros)/calib/pirvs.yaml" type="str" />

    <rosparam file="$(find svo_ros)/param/$(arg param).yaml" />

  </node>

  <!-- RVIZ + SVO GUI -->
  <node name="vis" pkg="rviz" type="rviz" args=" -d $(find svo_ros)/rviz_config.rviz" />
  <node name="svo_gui" pkg="rqt_gui" type="rqt_gui" args="-s rqt_svo.svo.Svo --args --topic svo" />
</launch>
